src/utils/gaa.js

```diff
@@ -369,6 +357,16 @@ export class GaaMeteringRegwall {
    * @param {{ iframeUrl: string }} params
    */
   static render_({iframeUrl}) {
+    const languageCode = getLanguageCodeFromElement(self.document.body);
+
+    // Tell the iframe which language to render.
```

Comment:
Prefer to use `addQueryParam` here: https://github.com/subscriptions-project/swg-js/blob/41f226737a782a2afd56d312ff7f093b804cdb9f/src/utils/url.js#L190

Reply:
Good call, updated